specs/latest/2.0/index.html

```diff
@@ -968,12 +967,12 @@ <h3><a name="WEBGLRENDERINGCONTEXT">The WebGL context</a></h3>
   [WebGLHandlesContextLoss] GLboolean isVertexArray(WebGLVertexArrayObject? vertexArray);
   void bindVertexArray(WebGLVertexArrayObject? array);
 };
-WebGL2RenderingContextBase implements WebGLRenderingContextBase;
+WebGL2RenderingContextBase includes WebGLRenderingContextBase;
```

Comment:
So mixins cannot include other mixins. Instead you need to write:

```webidl
interface WebGL2RenderingContext {};
WebGL2RenderingContext includes WebGLRenderingContextBase; WebGL2RenderingContext includes WebGL2RenderingContextBase;
```

Reply:
Ah, you're right. I'll make a follow-up PR.